iotedgedev/cli.py

```diff
@@ -423,6 +423,25 @@ def azure(setup,
             envvars.save_envvar("DEVICE_CONNECTION_STRING", envvars.DEVICE_CONNECTION_STRING)
             output.info("Updated current .env file")
 
+# @click.command(context_settings=CONTEXT_SETTINGS, help="Create IoT Edge Modules")
```

Comment:
This is just for temporary testing purpose. I will revisit the CLI entry point design later.

Reply:
There are two API surfaces due to legacy reasons. 

`module --create` follows the old pattern

`iotedgedev createmodule` follows the new pattern


See in cli.py how I have both for say iotedgedev build and iotedgedev modules --build, first one calls the second one.

Reply:
Thanks. I have added the new style one. In addition, the command is renamed from "create module" to "add module" for simplicity and aligning with VS Code extension's wording. I will update README once we finalize the API.